msal/src/main/java/com/microsoft/identity/client/internal/api/BrokerClientIdRefreshTokenAccessor.java

```diff
@@ -59,18 +79,12 @@
      * @return an RT, if there's any.
      * @throws MsalClientException if the calling app is not a broker app.
      */
-    public static @Nullable String get(@NonNull final Context context,
-                                       @NonNull final String accountObjectId) throws MsalClientException {
+    public static @Nullable
+    String get(@NonNull final Context context,
+               @NonNull final String accountObjectId) throws MsalClientException {
         final String methodName = "getBrokerRefreshToken";
```

Comment:
> methodName = "getBrokerRefreshToken" [](start = 20, length = 37)

 isn't the method name "get" here or "BrokerClientIdRefreshTokenAccessor.get". 
Any idea why it is set as  "getBrokerRefreshToken"

Reply:
No idea...probably a typo here

Reply:
or may be a pattern that we have been following that I'm not aware of

Reply:
I'll keep this is as is for now....we can fix it later in a separate PR if needed as I believe we've followed this pattern in a number of places